for BZ1647168 -- small change to cherry pick to enterprise-3.11

jhoyt-rh opened the pull request:
New PR for branch based on master instead of previous enterprise-3.10. Needed to create new branch in order to cherrypick to 3.11 and get content in master.

BZ is here: https://bugzilla.redhat.com/show_bug.cgi?id=1647168

Previous PR is here: https://github.com/openshift/openshift-docs/pull/13438

vikram-redhat commented:
Hi @jhoyt-rh - maybe I am missing it, but the changes required by the bug are not in this PR. This PR only contains white space changes and a simple one line change?

The changes required for this bug were in this PR: https://github.com/openshift/openshift-docs/pull/13089/files. Where are those changes? The PR that you have linked (#13438) is for the fix from .html to .adoc, which was a mistake that came out of the original PR #13089.

jhoyt-rh commented:
@vikram-redhat

Closing this PR due to errors. I have corrected those errors that Vikram pointed out and opened a new PR. That new PR is here: https://github.com/openshift/openshift-docs/pull/13515

I have removed all labels and milestones from this PR prior to closing.
